client/client.go

```diff
@@ -183,9 +183,9 @@ func (cli *Client) Close() error {
 
 // getAPIPath returns the versioned request path to call the api.
 // It appends the query parameters to the path if they are not empty.
-func (cli *Client) getAPIPath(p string, query url.Values) string {
+func (cli *Client) getAPIPath(p string, query url.Values, ignoreVersion bool) string {
```

Comment:
and by that I mean:

`Client.Ping` -> `Client.get`
`Client.get` -> `Client.sendRequest`
`Client.sendRequest` -> `Client.sendClientRequest`
`Client.sendClientRequest` -> `Client.newRequest`
`Client.newRequest` -> `Client.getAPIPath`

The best I could see is to add a bool to newRequest and getAPIPath and do:

`Client.Ping` -> `Client.sendClientRequestIgnoreVersion`
`Client.sendClientRequestIgnoreVersion` -> `Client.newRequest(ignoreVersion=true)`
`Client.newRequest` -> `Client.getAPIPath(ignoreVersion=true)`


wdyt @dnephin @vdemeester